package.json

```diff
@@ -60,17 +60,18 @@
     "electron-log": "4.3.2",
     "electron-squirrel-startup": "1.0.0",
     "electron-store": "5.1.1",
+    "faker": "5.5.3",
     "generate-password": "1.6.0",
     "hex-string": "1.0.3",
     "history": "4.10.1",
     "http": "0.0.1-security",
     "ini": "2.0.0",
-    "faker": "5.5.3",
     "javascript-terminal": "1.1.1",
     "jpg-glitch": "pastelnetwork/jpg-glitch#53a0839",
     "js-base64": "3.5.2",
     "js-sha256": "0.9.0",
     "libsodium-wrappers-sumo": "0.7.6",
+    "moment": "2.29.1",
```

Comment:
Please remove moment for approval (I hit wrong button so adding as a separate comment)

Reply:
I use the moment for calculating the date(can't implement using datejs). This is necessary for me. The datejs is not good than moment.